.jenkins/scripts/release.sh

```diff
@@ -20,9 +20,13 @@ set -xe
 # Parameters:
 # $1: Branch
 # $2: current version
+# $3: extra build args for all mvn cmd
 main() {
   local branch="${1?Missing branch}"
   local currentVersion="${2?Missing project version}"
+  local extraBuildParams="${3}"
+
+
```

Comment:
why empty lines?

Reply:
It was to add space between parameters vars and others vars